helm-chart/dash0-operator/README.md

```diff
@@ -727,6 +727,36 @@ resource attribute auto-detection.
 This allows Dash0 users to avoid the hassle of manually adding the OpenTelemetry SDK to their applications.
 Dash0 takes care of it automatically.
 
+### Specifying additional resource attributes via pod labels and annotations
+
+**Important:** All the labels and annotations listed above must be specified at the level of the pod spec.
+The labels and annotations specified below will be ignored when set, for example, in the metadata of deployments and daemonsets, as opposed  to the metadata of their respective pod templates.
+
+The following [standard Kubernetes labels](https://kubernetes.io/docs/reference/labels-annotations-taints/#labels-annotations-and-taints-used-on-api-objects) are mapped toi resource attributes as follows:
+
+* `app.kubernetes.io/name` => `service.name`
+* `app.kubernetes.io/version` => `service.version`
+* `app.kubernetes.io/part-of` => `service.namespace`
+* `app.kubernetes.io/instance` => `service.instance.id`
+
+Note: the `OTEL_SERVICE_NAME` environment variable will have precedence over the `app.kubernetes.io/name` label.
```

Comment:
I imagine pre-set OTEL_RESOURCE_ATTRIBUTES take precendence as well?

Reply:
yes

Reply:
To clarify: The intention of my comment was that we should also spell that out explicitly here.

Reply:
Done in a4cc1d3